tasks/main.yml

```diff
@@ -51,6 +51,17 @@
   with_items: "{{ backups_role_db_names }}"
   when: _backups_role_postgresql_enabled
 
+- name: Grant USAGE priv on public schema
+  postgresql_privs:
+    db: "{{ backups_role_db_names[0] }}"
```

Comment:
this is only the DB to connect to run the `ALTER TABLE` command so we can just pick the first one on the list.